### PR TITLE
[FrameworkBundle][Configuration] Fix translator enabled_locales configuration definition

### PR DESCRIPTION
**Pull request by fancyweb**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -688,7 +688,7 @@ private function addTranslatorSection(ArrayNodeDefinition $rootNode)
                             ->prototype('scalar')->end()
                         ->end()
                         ->arrayNode('enabled_locales')
-                            ->prototype('scalar')
+                            ->prototype('scalar')->end()
                             ->defaultValue([])
```
Comment (stof) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php]: `->defaultValue([])` could even be removed entirely here. It is already the default value for a prototyped array node. We don't need to move the broken default value to the `enabled_locales` node.

**Review by stof — CHANGES_REQUESTED** [review]

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php (hunk 1 above)
Comment (fancyweb) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php]: We have a lot of occurences of `->defaultValue([])`on prototyped array nodes. Couldn't we clean them all starting from a lower branch instead?

Comment (fabpot) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php]: Let's do that in another PR indeed.

**Review by fabpot — APPROVED** [review]

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php (hunk 1 above)
Comment (stof) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php]: Well, but here, you are actually adding it on that node (as the code was previously on the scalar node).

**Review by yceruto — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @fancyweb.

**Pull request merged**